### ISSUE TITLE
elm npm package graceful-fs depecrated WARN message

### ISSUE THREAD

**Issue by nullbio** [elm/compiler#1790]
I'm constantly getting this error:

```
└[~/g/s/g/v/t/front]> npm update -g
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
```
Because the elm npm package relies on `binwrap 0.14.0` which has dependencies that rely on an old version of graceful-fs. 

By updating the elm npm package package.json to point to the new version of binwrap (0.2.0) it should fix this error, because the new binwrap's dependencies point to the newer version of graceful-fs.

**Comment by RonaldTinashe** [comment]
In the meantime, are there any known workarounds for this?

**Comment by evancz** [comment]
We recently rewrote the `npm` install script to not have any dependencies at all. That seemed like the easiest way to make it so users are never faced with information like this when all they want is a binary downloaded from https://github.com/elm/compiler/releases/tag/0.19.0

The new version is at `0.19.0-no-deps` and a slightly improved version should be available when 0.19.1 comes out. (Better on Windows!)
